tools/offcputime.py

```diff
@@ -183,24 +140,17 @@ def signal_ignore(signal, frame):
     if not folded:
         print()
     counts = b.get_table("counts")
+    stack_traces = b.get_table("stack_traces")
     for k, v in sorted(counts.items(), key=lambda counts: counts[1].value):
         if folded:
             # print folded stack output
-            line = k.name.decode() + ";"
-            for i in reversed(range(0, maxdepth)):
-                if k.ret[i] == 0:
-                    continue
-                line = line + b.ksym(k.ret[i])
-                if i != 0:
-                    line = line + ";"
```

Comment:
Same here -- we were dropping the topmost frame (which ought to always be `finish_task_switch`). Shall I drop it too?